docs/deploy/ubuntu_pro.rst

```diff
@@ -0,0 +1,26 @@
+Ubuntu Pro
+==========
+
+`Ubuntu Pro <https://ubuntu.com/pro>`__ is an enterprise subscription from Canonical Ltd that provides (among other features) extended security support after the standard end-of-life date, this is also referred to as Extended Security Maintenance (ESM).
+
+.. note::
+
+   `Salt Stack does not officially support ESM<https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-supported-operating-systems.html#ubuntu>`__, an end-of-life server will need removing from the Deploy repo regardless.
+
+Installation
+------------
+
+#. Purchase an Ubuntu Pro licence from `Canonical <https://ubuntu.com/pro/subscribe>`__.
```

Comment:
We cannot purchase individual licences for VPS servers (Canonical only provide these in bulk), so this process is only for hardware servers.